app/router.js

```diff
@@ -6,6 +6,7 @@ var Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.resource('login');
```

Comment:
Can you remind me again what you told us about the difference between route/resource? I think it was about verbs/nouns, so I'm assuming the reasoning here is that login is a verb?